.github/CPS-TEMPLATE.md

```diff
@@ -7,7 +7,7 @@ Authors:
     - John Doe <john.doe@email.domain>
 Proposed Solutions: []
 Discussions:
-    - https://github.com/cardano-foundation/cips/pulls/?
+    - https://github.com/cardano-foundation/cips/pull/?
```

Comment:
```suggestion
    - https://github.com/cardano-foundation/CIPs/pull/?
```

Reply:
grrr; I'd fixed one but forgot the other...